src/app/page.tsx

```diff
@@ -1,9 +1,166 @@
-import styles from './page.module.css';
+'use client';
+
+import COLORS from '@/styles/colors';
+import Image from 'next/image';
+import { H2, H3, H4 } from '../styles/text';
+import { LinkButton } from '../components/Button';
+
+import {
+  TitleSection,
+  PageContainer,
+  ButtonDiv,
+  StatisticsSection,
+  StatLabel,
+  StatContainer,
+  HowYouCanHelpContainer,
+  Title,
+  ServiceContainer,
+  ServiceIcon,
+  ServicesDiv,
+  MissionValuesContainer,
+  MissionStatement,
+  TextContainer,
+  ImageBackground,
+  ImageContainer,
+  ImageWrapper,
+} from './styles';
+
+type Stat = {
+  number: string;
+  label: string;
+};
+
+const stats: Stat[] = [
+  { number: '100+', label: 'Matched Cases' },
+  { number: '50+', label: 'Attorneys' },
+  { number: '100+', label: 'Translation Support' },
+  { number: '100+', label: 'Successful Outcomes' },
+];
+
+const renderStat = (stat: Stat) => (
+  <StatContainer key={stat.label}>
+    <H2 $color={COLORS.blueDark}>{stat.number}</H2>
+    <StatLabel>{stat.label}</StatLabel>
+  </StatContainer>
+);
+type Service = {
+  iconSrc: string;
+  title: string;
+  description: string;
+};
+
+const services: Service[] = [
+  {
+    iconSrc: 'An Icon will be Here',
+    title: 'Case Assistance',
+    description:
+      'Attorneys licensed in any state with or without immigration law experience can represent immigrants and asylum-seekers in removal proceeding before the Immigration Court, the Board of Immigration Appeals and the U.S. Court of Appeals for the Ninth Circuit.',
+  },
+  {
+    iconSrc: 'An Icon will be Here',
+    title: 'Limited Case Assistance',
+    description:
+      'Law students and recent graduates awaiting bar results can assist attorneys to screen potential clients, conduct legal and factual research and write motions and briefs in support of on-going court cases.',
+  },
+  {
+    iconSrc: 'An Icon will be Here',
+    title: 'Translation Assignment',
+    description:
+      'Interpreters and translators can conduct volunteer translation and live interpertation. We have frequent need for individuals who speak Creole, Portuguese, Spanish, French, Arabic, and Russian',
+  },
+];
+
+const renderService = (service: Service) => (
+  <ServiceContainer key={service.title}>
+    <ServiceIcon src={service.iconSrc} alt={service.title} />
+    <H3 $color={COLORS.blueMid}>{service.title}</H3>
+    <H4 $color={COLORS.greyDark} $fontWeight="400">
+      {service.description}
+    </H4>
```

Comment:
I think it may be better to use `<H4 ... dangerouslySetInnerHTML={{ __html: service.description }} />` and change the `description` field to be a JSX element (i.e. `<><b>Interpreters and translators</b> can ...</>`)

This will allow us to match the bolded words in Figma and allow us to use the generalization method you have implemented here.